custom.properties.example

```diff
@@ -0,0 +1,12 @@
+DEFAULT_IDENTITY_POOL_ID=xx-xxxx-x:xxxx-xxxx-xxxx-xxxx
+DEFAULT_REGION=xx-xxxx-x
+DEFAULT_SE_IDENTITY_POOL_ID=xx-xxxx-x:xxxx-xxxx-xxxx-xxxx
```

Comment:
What does SE stand for? Can you come up with more descriptive name or adding comment for these 2 new properties (DEFAULT_SE_IDENTITY_POOL_ID, and DEFAULT_SE_REGION) in follow up PR. 
I'll merge it for now to unblock deployment and e2e test.